server/pulp/server/async/scheduler.py

```diff
@@ -6,18 +6,20 @@
 import platform
 import threading
 import time
+import uuid
+import mongoengine
```

Comment:
Also within each section imports should be alphabetized so `mongoengine` would come before `uuid`.